tests/fast-integration/eventing-test/metric-test.js

```diff
@@ -200,6 +200,11 @@ function runDashboardTestCase(dashboardName, test) {
 }
 
 async function eventingMonitoringTest(backend, isSkr, isJetStreamEnabled = false) {
+  if (isSkr) {
```

Comment:
Do we have a follow-up ticket for this ? If yes, it would be nice to link it here :)

Reply:
yes